python/setup.py

```diff
@@ -98,6 +98,9 @@ def run(self):
                       'build type (debug or release), default release'),
                      ('boost-namespace=', None,
                       'namespace of boost (default: boost)'),
+                     ('with-arrow-gpu', None, 'build the Arrow GPU extension'),
+                     ('without-arrow-gpu', None,
```

Comment:
Uh... No need for two contradictory options. Just keep `with-arrow-gpu`.

Reply:
When one builds pyarrow without `--with-arrow-gpu` option and does not define `PYARROW_WITH_ARROW_GPU` environment variable, what should be the default behavior when libarrow_gpu library is available?
Enable `--with-arrow-gpu` automatically (current behavior, hence the `--without-arrow-gpu` option to override the default behaviour) or always require explicit usage of `--with-arrow-gpu` (or the environment variable)?

Reply:
AFAIK, other options (Plasma, Parquet, Orc) are enabled automatically.

Reply:
`with-cuda`, per above comments

Reply:
Replaced `--with/without-arrow-gpu` with `--with-cuda`.